narwhal/executor/src/metrics.rs

```diff
@@ -38,6 +40,23 @@ pub struct ExecutorMetrics {
     /// Latency between the time when the batch has been
     /// created and when it has been fetched for execution
     pub batch_execution_latency: Histogram,
+    /// The number of batches per committed subdag to be fetched
+    pub committed_subdag_batch_count: Histogram,
+    /// Latency for time taken to fetch all batches for committed subdag
+    /// either from local or remote worker.
+    pub batch_fetch_for_committed_subdag_total_latency: Histogram,
+    /// The number of successful local batch fetches
+    pub successful_local_batch_fetch: IntCounter,
```

Comment:
Normally I see these sort of metrics done as one "counter" metric with a "status" field, might be simpler here?

So you'd have just a single metric, `batch_fetches` with labels `source`->local/remote, and `status`->success/failure/timeout.